### ISSUE TITLE
Intermediate result show N/A

### ISSUE THREAD

**Issue by gungui98** [microsoft/nni#3118]
**Environment**:
- NNI version: 1.9
- NNI mode (local|remote|pai): local
- Client OS: Ubuntu 18.04
- Server OS (for remote mode only):
- Python version: 3.6
- PyTorch/TensorFlow version: Pytorch 1.4.0
- Is conda/virtualenv/venv used?: conda
- Is running in Docker?: No

**Log message**:

**What issue meet, what's expected?**:
when I run the experiment, the intermediate result chart hover always shows N/A until the trials complete. This also happened when I cancel the trial

**Additional information**:
![Untitled](https://user-images.githubusercontent.com/22836931/100078490-282d2d00-2e76-11eb-96bc-528dccd13a84.png)

**Comment by Lijiaoa** [comment]
@gungui98 
Please help download the experiment results and upload data for debug. https://github.com/microsoft/nni/blob/master/docs/img/webui-img/download.png

**Comment by gungui98** [comment]
### The NNI Dispatcher log:

```
[11/25/2020, 09:17:45 AM] INFO (hyperopt.utils/MainThread) Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
[11/25/2020, 09:17:45 AM] INFO (hyperopt.fmin/MainThread) Failed to load dill, try installing dill via "pip install dill" for enhanced pickling support.
[11/25/2020, 09:17:45 AM] INFO (nni.msg_dispatcher_base/MainThread) Start dispatcher
[11/25/2020, 09:17:45 AM] INFO (hyperopt.tpe/Thread-1) tpe_transform took 0.002926 seconds
[11/25/2020, 09:17:45 AM] INFO (hyperopt.tpe/Thread-1) TPE using 0 trials
[11/25/2020, 09:17:45 AM] INFO (hyperopt.tpe/Thread-1) tpe_transform took 0.001230 seconds
[11/25/2020, 09:17:45 AM] INFO (hyperopt.tpe/Thread-1) TPE using 0 trials
```

### NNIManager log

```
[11/25/2020, 9:17:44 AM] INFO [ 'Datastore initialization done' ]
[11/25/2020, 9:17:44 AM] INFO [ 'RestServer start' ]
[11/25/2020, 9:17:44 AM] INFO [ 'Construct local machine training service.' ]
[11/25/2020, 9:17:44 AM] INFO [ 'RestServer base port is 8090' ]
[11/25/2020, 9:17:44 AM] INFO [ 'Rest server listening on: http://0.0.0.0:8090' ]
[11/25/2020, 9:17:45 AM] INFO [ 'NNIManager setClusterMetadata, key: local_config, value: {"useActiveGpu":true,"gpuIndices":"0,1,2,3"}' ]
[11/25/2020, 9:17:45 AM] INFO [ 'Specified GPU indices: 0,1,2,3' ]
[11/25/2020, 9:17:45 AM] INFO [ 'NNIManager setClusterMetadata, key: trial_config, value: {"command":"python train_ner.py","codeDir":"/home/cotai/phi/fast_bert/./","gpuNum":4}' ]
[11/25/2020, 9:17:45 AM] INFO [ 'required GPU number is 4' ]
[11/25/2020, 9:17:45 AM] INFO [ 'Starting experiment: S57ZNnKj' ]
[11/25/2020, 9:17:45 AM] INFO [ 'Change NNIManager status from: INITIALIZED to: RUNNING' ]
[11/25/2020, 9:17:45 AM] INFO [ 'Add event listeners' ]
[11/25/2020, 9:17:45 AM] INFO [ 'Run local machine training service.' ]
[11/25/2020, 9:17:45 AM] WARNING [ 'gpu_metrics file does not exist!' ]
[11/25/2020, 9:17:45 AM] INFO [ 'NNIManager received command from dispatcher: ID, ' ]
[11/25/2020, 9:17:45 AM] INFO [ 'NNIManager received command from dispatcher: TR, {"parameter_id": 0, "parameter_source": "algorithm", "parameters": {"max_seq_length": 32, "train_batch_size": 16, "num_train_epochs": 16, "learning_rate": 1e-05}, "parameter_index": 0}' ]
[11/25/2020, 9:17:45 AM] INFO [ 'NNIManager received command from dispatcher: TR, {"parameter_id": 1, "parameter_source": "algorithm", "parameters": {"max_seq_length": 256, "train_batch_size": 16, "num_train_epochs": 32, "learning_rate": 0.0007}, "parameter_index": 0}' ]
[11/25/2020, 9:17:50 AM] INFO [ 'submitTrialJob: form: {"sequenceId":0,"hyperParameters":{"value":"{\\"parameter_id\\": 0, \\"parameter_source\\": \\"algorithm\\", \\"parameters\\": {\\"max_seq_length\\": 32, \\"train_batch_size\\": 16, \\"num_train_epochs\\": 16, \\"learning_rate\\": 1e-05}, \\"parameter_index\\": 0}","index":0}}' ]
[11/25/2020, 9:17:50 AM] INFO [ 'submitTrialJob: form: {"sequenceId":1,"hyperParameters":{"value":"{\\"parameter_id\\": 1, \\"parameter_source\\": \\"algorithm\\", \\"parameters\\": {\\"max_seq_length\\": 256, \\"train_batch_size\\": 16, \\"num_train_epochs\\": 32, \\"learning_rate\\": 0.0007}, \\"parameter_index\\": 0}","index":0}}' ]
[11/25/2020, 9:17:55 AM] INFO [ 'Trial job oaMs0 status changed from WAITING to RUNNING' ]
[11/25/2020, 9:21:10 AM] INFO [ 'Trial job oaMs0 status changed from RUNNING to SUCCEEDED' ]
[11/25/2020, 9:21:10 AM] INFO [ 'NNIManager received command from dispatcher: TR, {"parameter_id": 2, "parameter_source": "algorithm", "parameters": {"max_seq_length": 63, "train_batch_size": 16, "num_train_epochs": 16, "learning_rate": 0.000148}, "parameter_index": 0}' ]
[11/25/2020, 9:21:15 AM] INFO [ 'Trial job vfmXX status changed from WAITING to RUNNING' ]
[11/25/2020, 9:21:15 AM] INFO [ 'submitTrialJob: form: {"sequenceId":2,"hyperParameters":{"value":"{\\"parameter_id\\": 2, \\"parameter_source\\": \\"algorithm\\", \\"parameters\\": {\\"max_seq_length\\": 63, \\"train_batch_size\\": 16, \\"num_train_epochs\\": 16, \\"learning_rate\\": 0.000148}, \\"parameter_index\\": 0}","index":0}}' ]
```
### Experiment parameters
```
{
    "experimentParameters": {
        "id": "S57ZNnKj",
        "revision": 51,
        "execDuration": 455,
        "logDir": "/home/cotai/nni-experiments/S57ZNnKj",
        "nextSequenceId": 5,
        "params": {
            "authorName": "default",
            "experimentName": "bert_ner",
            "trialConcurrency": 2,
            "maxExecDuration": 18000,
            "maxTrialNum": 10,
            "searchSpace": {
                "max_seq_length": {
                    "_type": "choice",
                    "_value": [
                        32,
                        63,
                        128,
                        256
                    ]
                },
                "train_batch_size": {
                    "_type": "choice",
                    "_value": [
                        4,
                        8,
                        16
                    ]
                },
                "num_train_epochs": {
                    "_type": "choice",
                    "_value": [
                        4,
                        8,
                        16,
                        32
                    ]
                },
                "learning_rate": {
                    "_type": "choice",
                    "_value": [
                        0.00001,
                        0.000079,
                        0.000148,
                        0.000217,
                        0.000286,
                        0.000355,
                        0.000424,
                        0.000493,
                        0.000562,
                        0.000631,
                        0.0007
                    ]
                }
            },
            "trainingServicePlatform": "local",
            "tuner": {
                "builtinTunerName": "TPE",
                "gpuIndices": "0,1,2,3",
                "classArgs": {
                    "optimize_mode": "maximize"
                },
                "checkpointDir": "/home/cotai/nni-experiments/S57ZNnKj/checkpoint"
            },
            "versionCheck": true,
            "clusterMetaData": [
                {
                    "key": "codeDir",
                    "value": "/home/cotai/phi/fast_bert/./"
                },
                {
                    "key": "command",
                    "value": "python train_ner.py"
                }
            ]
        },
        "startTime": 1606270665227
    },
    "trialMessage": []
}
```

**Comment by Lijiaoa** [comment]
oh, I'm sorry to find that `"trialMessage": []` is empty. `"trialMessage": []` should store all trial's intermediate and final results, it is a bug because we refactor code in v1.9,
thanks for helping find this.

And about your hover NaN bug, please copy these data`(http://ip:port/api/v1/nni/trial-jobs and http://ip;port/api/v1/nni/metric-data)` to a file or two file. Thanks.
